here/traffic/sql/dynamic_bins/create-table-congestion_raw_corridors.sql

```diff
@@ -0,0 +1,80 @@
+-- Table: gwolofs.congestion_raw_corridors
+
+-- DROP TABLE IF EXISTS gwolofs.congestion_raw_corridors;
+
+CREATE TABLE IF NOT EXISTS gwolofs.congestion_raw_corridors
+(
+    corridor_id smallint,
+    time_grp timerange NOT NULL,
+    bin_range tsrange NOT NULL,
+    tt numeric,
```

Comment:
<img width="394" height="419" alt="Image" src="https://github.com/user-attachments/assets/b1d887f8-435e-4000-ab65-c8fb97e64345" />

The precision on these is wiiild. I'd suggest `real`s here too to save space without losing any meaningful precision.